projects/plugins/jetpack/modules/wordads/class-wordads.php

```diff
@@ -738,6 +743,12 @@ public function get_ad_snippet( $section_id, $height, $width, $location = '', $c
 			}
 
 			$loc_id = esc_js( $loc_id );
+
+			$format = WordAds_Formats::get_format_slug( (int) $width, (int) $height );
```

Comment:
This line be better to ensure `$format` is a string so that you don't need to convert it later.

Reply:
I left a related comment https://github.com/Automattic/jetpack/pull/43476#discussion_r2126683419.

The method get_format_slug returns either the Gutenberg format slug we support as an Ad placement or an empty string. So I decided that "not having" an empty string means we have found a right format.